GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -198,7 +198,11 @@ public void Stop()
             lock (this.postFetchJobLock)
             {
                 // TODO(Mac): System.PlatformNotSupportedException: Thread abort is not supported on this platform
-                this.postFetchJobThread?.Abort();
+
+                if (!GVFSPlatform.Instance.IsUnderConstruction)

```

Comment:
Can we just clean this up with a cross-plat solution now? Would it be more complex than setting a flag so the thread can terminate itself?

Reply:
The thread spends most of its time waiting for Git to return with a result. We can't watch a flag at the same time without creating _yet another thread_ to watch the flag and then somehow kill the other thread. Did you have a real solution in mind?

Reply:
In that case the solution seems to be to terminate the Git process. Every Windows-specific behavior we write is technical debt that we have to come back to, so I'd prefer to see a complete fix unless there's a strong argument against it.

Reply:
In fact even on Windows that would be the preferred behavior. It seems that today we are killing our own thread but orphaning the Git process for however long it continues to run.

Reply:
I added #409 to track your concerns here.